gemfiles/jwt_1.5.6.gemfile.lock

```diff
@@ -0,0 +1,55 @@
+PATH
```

Comment:
Perhaps we don't need to check in the lock files. Do we care if gems other than `jwt` get upgraded?

Reply:
I was thinking that too but appraisal's readme says:

> ... These are stored in the gemfiles directory, and should be added to version control to ensure that the same versions are always used.

https://github.com/thoughtbot/appraisal#under-the-hood

Unlike `Gemfile.lock`, it doesn't affect applications which depend on it.

Reply:
I also see [this](https://github.com/thoughtbot/appraisal#version-control).

> When using Appraisal, we recommend you check in the Gemfiles that Appraisal generates within the gemfiles directory, but exclude the lockfiles there (`*.gemfile.lock`.)

Reply:
Ah ok cool. Updated: 93fe319